html/login/.env

```diff
@@ -0,0 +1,4 @@
+DB_SERVERNAME=mysql-server
```

Comment:
why is this included in this repo?

.env should never be on the repo because your are going to expose private credentials

Reply:
you could use .gitignore file to prevent this from adding the .env file